Libraries/CMSIS/Device/Maxim/MAX32657/Include/system_max32657.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
          Update if use of this oscillator requires precise timing.*/
 /* NOTE: INRO was previously named NANORING */
 #ifndef INRO_FREQ
-#define INRO_FREQ 100000
+#define INRO_FREQ 131000
```

Comment:
Is this the nominal value that's indicated in the datasheet? The INRO frequency is highly variable which could be 131KHz on one part or 80KHz on another.

Reply:
Yes, it is given as 131kHz in both user guide and datasheet for MAX32657, if that is what you mean.